hw/acpi/aml-build.c

```diff
@@ -2360,7 +2342,186 @@ void acpi_dsdt_add_pci_bus(Aml *table, AcpiPciBus *pci_host)
     aml_append(method, aml_release(aml_name("BLCK")));
     aml_append(method, aml_return(aml_int(0)));
     aml_append(scope, method);
+}
```

Comment:
Were any of the functions modified in anyway or just moved over?

Reply:
simply moved over